README.md

```diff
@@ -683,206 +927,76 @@ Returns an array where matching items are filtered.
   // output: [1, 3]
   ```
 
-#### Browser Support
+#### Browser Support for `Array.prototype.filter()`
 
-![Chrome][chrome-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
-:-: | :-: | :-: | :-: | :-: |
-  1.0 ✔  |  1.5 ✔ |  9 ✔ |  ✔ |  ✔  |
+![Chrome][chrome-image] | ![Edge][edge-image] | ![Firefox][firefox-image] | ![IE][ie-image] | ![Opera][opera-image] | ![Safari][safari-image]
+:-: | :-: | :-: | :-: | :-: | :-: |
+  1.0 ✔  | ✔ | 1.5 ✔ |  9 ✔ |  ✔ |  ✔  |
 
 **[⬆ back to top](#quick-links)**
 
-### _.intersection
-Returns an array that is the intersection of all the arrays. Each value in the result is present in each of the arrays.
+## Collection*
```

Comment:
If you want to add new methods, do them in a new request.

We try to keep it to one change of method per PR